src/import/importConfiguration.ts

```diff
@@ -91,7 +91,8 @@ export function importConfiguration(yaml: YAMLConfiguration | string, file: stri
     contained: yaml.contained,
     extension: yaml.extension,
     modifierExtension: yaml.modifierExtension,
-    url: yaml.url, // minimum config property
+    canonical: yaml.canonical, // minimum config property
+    url: yaml.url ?? `${yaml.canonical}/ImplementationGuide/${yaml.id}`,
```

Comment:
And if you change the order there, could it change here too!

Reply:
If you give a mouse a cookie!

Reply:
But also yes.

Reply:
I think I'm going to move it to the top of the list because it seems odd to bury the required property at the bottom of the long list. That does put it above `id`, which seems a little bold, so let me know if you'd prefer it at the bottom.

Reply:
Top is good with me!